lib/libtsunami/src/dsapp.cpp

```diff
@@ -56,15 +62,28 @@ void DSApp::doAppFrame()
 
 void DSApp::visualPerFrame()
 {
-	m_scene->subRemoveFinished();
+	if (VideoMustLock()) {
```

Comment:
`visualPerFrame` called only in video thread, so this condition never happen. I think it's unnecessary here.